functions/src/user.ts

```diff
@@ -0,0 +1,46 @@
+import * as admin from "firebase-admin";
+import * as functions from "firebase-functions";
+import { HttpsError } from "firebase-functions/v1/https";
+
+import { HttpsFunctionHandler } from "./types/utility";
+import { throwErrorIfNotWorldOwner } from "./utils/permissions";
+import { getWorldBySlug } from "./utils/world";
+
+export const makeUserWorldOwner: HttpsFunctionHandler<{
+  worldSlug: string | undefined;
```

Comment:
Branded maybe? Does it make sense?

```suggestion
  worldSlug?: WorldSlug | string;
```

Reply:
We don't yet have `branded` in function. I wonder if it's worth adding this concept here given that we can't reuse it from our core codebase. @colinhowe wdyt?